frontend/src/metabase/visualizations/components/Legend.module.css

```diff
@@ -34,6 +34,6 @@ fullscreen dashboard mode
   margin-top: 1em;
 }
 
-.DashCard :global(.LegendItem) {
+.DashCard.LegendItem {
```

Comment:
`DashboardGrid` (from `frontend/src/metabase/dashboard/components/DashboardGrid.tsx`) already uses the `LegendS.DashCard` selector so this change should be as simple as converting the `LegendItem` class to the CSS selector